docker-compose.yml

```diff
@@ -1,26 +1,32 @@
-version: '2.1'
+---
+version: '2.4'
 
 services:
-  cpo-case-payment-orders-api:
-    build:
-      context: .
-      args:
-        - http_proxy
-        - https_proxy
-        - no_proxy
-    image: hmctspublic.azurecr.io/spring-boot/template
+  case-payment-orders-api:
+    image: "hmctspublic.azurecr.io/cpo/case-payment-orders-api:latest"
+    container_name: case-payment-orders-api
     environment:
-      # these environment variables are used by java-logging library
-      - ROOT_APPENDER
-      - JSON_CONSOLE_PRETTY_PRINT
-      - ROOT_LOGGING_LEVEL
-      - REFORM_SERVICE_TYPE
-      - REFORM_SERVICE_NAME
-      - REFORM_TEAM
-      - REFORM_ENVIRONMENT
-      - LOGBACK_DATE_FORMAT
-      - LOGBACK_REQUIRE_THREAD
-      - LOGBACK_REQUIRE_ALERT_LEVEL=false
-      - LOGBACK_REQUIRE_ERROR_CODE=false
+      SPRING_DATASOURCE_URL: jdbc:postgresql://cpo-db:5432/cpo
+      CPO_S2S_AUTHORISED_SERVICES: xui_webapp
+      IDAM_OIDC_URL: https://idam-web-public.aat.platform.hmcts.net
+      IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
+      S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
     ports:
-      - $SERVER_PORT:$SERVER_PORT
+      - 4457:4457
+    depends_on:
+      - cpo-db
+
+  cpo-db:
+    image: postgres:11.11
+    restart: always
+    environment:
+      - POSTGRES_DB=cpo
```

Comment:
Should this correspond to the default in the `src/main/resources/application.yaml`, i.e. `cpo_case_payment_orders`?

Reply:
I just used all defaults as it is from application.yaml. We can externalise with params and define few more variables under environment section. But the frequency of changing these default values are very rare.

Reply:
Happy to change if parameterisation is required 😄

Reply:
Your value for `POSTGRES_DB` does not match [the equivalent in `application.yaml`](https://github.com/hmcts/cpo-case-payment-orders-api/blob/master/src/main/resources/application.yaml#L36) i.e. `cpo` =! `cpo_case_payment_orders`.

Reply:
not required it should match with the value in SPRING_DATASOURCE_URL.